run.py

```diff
@@ -13,59 +13,66 @@
 # limitations under the License.
 
 import logging
-
 from model import LanguageModelConfig, TransformerConfig, QuantizedWeight8bit as QW8Bit
 from runners import InferenceRunner, ModelRunner, sample_from_model
 
-
+# Path to the checkpoint directory
 CKPT_PATH = "./checkpoints/"
 
-
 def main():
+    # Initialize model configuration
     grok_1_model = LanguageModelConfig(
-        vocab_size=128 * 1024,
+        vocab_size=128 * 1024,  # 128K vocabulary size
         pad_token=0,
         eos_token=2,
-        sequence_len=8192,
+        sequence_len=8192,  # Sequence length
```

Comment:
Why add a comment that repeats the parameter name? Is there genuine concern that "sequence_len" isn't already clear?